python/src/geolibre/geolibre.py

```
@@ -976,6 +976,54 @@ def set_layer_opacity(self, layer: str | Layer, opacity: float) -> None:
+        self._update_project(_move)
+
+    def duplicate_layer(self, layer: str | Layer, *, name: str | None = None) -> str:
+        """Duplicate a layer, returning the new layer id."""
+        source = copy.deepcopy(self._resolve_layer(layer)._layer())
+        source["id"] = str(uuid.uuid4())
+        source["name"] = name or f"{source.get('name', 'Layer')} copy"
+        return self._add_layer(source)
+
+    def show_layer(self, layer: str | Layer) -> None:
+        """Show a layer."""
+        self.set_layer_visibility(layer, True)
```

Ruff (0.16.1)

[warning] Boolean positional value in function call

(FBT003)